go/tasks/pluginmachinery/flytek8s/copilot.go

```diff
@@ -62,6 +63,34 @@ func FlyteCoPilotContainer(name string, cfg config.FlyteCoPilotConfig, args []st
 	}, nil
 }
 
+func CopilotCommandArgs(storageConfig *storage.Config) []string {
+	var commands = []string{
+		"/bin/flyte-copilot",
+		"--storage.enable-multicontainer",
+		"--storage.limits.maxDownloadMBs=0",
+		fmt.Sprintf("--storage.type=%s", storageConfig.Type),
+	}
+	if !reflect.DeepEqual(storageConfig.Stow, storage.StowConfig{}) {
+		var cfg string
+		for key, val := range storageConfig.Stow.Config {
+			cfg += fmt.Sprintf("%s=%s,", key, val)
+		}
+		return append(commands, []string{
```

Comment:
should we end here? I think we should append all the options and let the storage library decide which ones take over the others?

Reply:
that true...I added the changes